router/pkg/config/config.schema.json

```diff
@@ -1860,6 +1860,11 @@
           "default": false,
           "description": "Omit the extensions field of Subgraph errors. If the value is true (default: false), the extensions field of Subgraph errors will be omitted. This is useful in case you want to avoid leaking internal information to the client. Some users of GraphQL leverage the errors.extensions.code field to implement error handling logic in the client, in which case you might want to set this to false."
         },
+        "omit_custom_fields": {
+          "type": "boolean",
+          "default": false,
+          "description": "Omit custom field of Subgraph errors in pass through mode. If the value is true (default: false), all fields except message and extensions will be omitted."
```

Comment:
What about locations field? Is it still propagated when set `omit_locations` false?

Reply:
Using allowlist instead